SuperteamDAO/earn
# feat: suggests for sponsors while creating a new listing

## What does this PR do?
Suggestions with external links to show similar listings which will help sponsors create listings faster

## Where should the reviewer start?

## How should this be manually tested?
Go to sponsor dashboard, create a new `Bounty Listing`
Enter 'Design' or 'Deep Dive' in listing title and focus out of the field (move to another field) 
and you'll be able to see the suggestions

## Any background context you want to provide?
/src/features/listing-buider/components/ListingBuilder/ListingBasic.ts

## What are the relevant issues?

## Screenshots (if appropriate)
